src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -122,12 +122,16 @@ public function load(array $configs, ContainerBuilder $container)
             foreach ($bundle['paths'] as $path) {
                 $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($path, $namespace));
             }
+
+            $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($config['default_path'].'/bundles/'.$namespace, $namespace));
```

Comment:
Btw, this path should be added between:
```php
if ($container->fileExists($dir = $container->getParameter('kernel.root_dir').'/Resources/'.$name.'/views', false)) {
    $bundleHierarchy[$name]['paths'][] = $dir;
}

// ---------> here <---------

if ($container->fileExists($dir = $bundle['path'].'/Resources/views', false)) {
    $bundleHierarchy[$name]['paths'][] = $dir;
}
```
Otherwise we can't able to override templates from third-party bundles, because the original will be loaded first.

Reply:
Also the path needs to be checked if it exists before add it, otherwise Twig throw an exception:
```bash
[Twig_Error_Loader]                                                                                                                      
  The "templates/bundles/Twig" directory does not exist ("/home/yceruto/github/symfony/symfony-demo/templates/bundles/Twig"). 
```